client/src/components/Navbar.js

```diff
@@ -40,160 +42,110 @@ const useStyles = (theme) => ({
 
 function Navbar(props) {
   const [redirect, setRedirect] = useState(null);
-
-  // logout button event handler
+  const [loggedIn, setLoggedIn] = useState(false);
+  const [profileImg, setProfileImg] = useState(null);
   const logout = (event) => {
     event.preventDefault();
 
     axios
       .post('/users/logout')
       .then((response) => {
         if (response.status === 200) {
-          props.updateUser({
-            loggedIn: false,
-            username: null,
-          });
+          setLoggedIn(false);
+          localStorage.clear();
           setRedirect('/');
+          window.location.reload();
```

Comment:
is this necessary? Does it not log out without a reload?

Reply:
Yes, I added that in purpose because when you do log out the cookie or current user id cleaned up but the navbar components still not changed.